tests/src/test/scala/system/health/BasicHealthTest.scala

```diff
@@ -47,141 +47,146 @@ import com.jayway.restassured.RestAssured
+  with Matchers
+  with WskActorSystem
+  with BeforeAndAfterAll
+  with TestHelpers
+  with WskTestHelpers
+  with JsHelpers {
+
+  val topic = "test"
+  val sessionTimeout = 10 seconds
+
+  implicit val wskprops = WskProps()
+  val wsk = new Wsk()
+
+  val messagingPackage = "/whisk.system/messaging"
+  val messageHubFeed = "messageHubFeed"
+  val messageHubProduce = "messageHubProduce"
+
+  val consumerInitTime = 10000 // ms
+
+  val kafkaUtils = new KafkaUtils
+
+  behavior of "Message Hub feed"
+
+  it should "create a new trigger" in withAssetCleaner(wskprops) {
+    val triggerName = s"newTrigger-${System.currentTimeMillis}"
+    println(s"Creating trigger ${triggerName}")
+
+    (wp, assetHelper) =>
+      val feedCreationResult = assetHelper.withCleaner(wsk.trigger, triggerName) {
+        (trigger, _) =>
+          trigger.create(triggerName, feed = Some(s"$messagingPackage/$messageHubFeed"), parameters = Map(
+            "user" -> kafkaUtils.getAsJson("user"),
+            "password" -> kafkaUtils.getAsJson("password"),
+            "api_key" -> kafkaUtils.getAsJson("api_key"),
+            "kafka_admin_url" -> kafkaUtils.getAsJson("kafka_admin_url"),
+            "kafka_brokers_sasl" -> kafkaUtils.getAsJson("brokers"),
+            "topic" -> topic.toJson
+          ))
+      }
+
+      withActivation(wsk.activation, feedCreationResult, initialWait = 5 seconds, totalWait = 60 seconds) {
+        activation =>
+          // should be successful
+          activation.response.success shouldBe true
+          val uuid = activation.response.result.get.fields.get("uuid").get.toString
+
+          // get /health endpoint and ensure it contains the new uuid
+          retry({
+            val response = RestAssured.given().get(System.getProperty("health_url"))
+            assert(response.statusCode() == 200 && response.asString().contains(uuid))
+          }, N = 3, waitBeforeRetry = Some(1.second))
+      }
+  }
+
+  it should "fire a trigger when a message is posted to message hub" in withAssetCleaner(wskprops) {
+    val currentTime = s"${System.currentTimeMillis}"
+
+    val baseTriggerName = "/_/BasicHealthTestTrigger"
+
+    val triggerName = System.getProperty("trigger.suffix") match {
+      case suffix if suffix != "" && suffix != null => s"${baseTriggerName}-${suffix}"
+      case _ => s"${baseTriggerName}-${currentTime}"
+    }
 
-    implicit val wskprops = WskProps()
-    val wsk = new Wsk()
+    (wp, assetHelper) =>
+      val result = wsk.trigger.get(triggerName, expectedExitCode = DONTCARE_EXIT)
 
-    val messagingPackage = "/whisk.system/messaging"
-    val messageHubFeed = "messageHubFeed"
-    val messageHubProduce = "messageHubProduce"
+      if (result.exitCode == NOT_FOUND) {
+        // trigger does not yet exist, create it
+        println(s"Creating trigger ${triggerName}")
 
-    val consumerInitTime = 10000 // ms
+        val feedCreationResult = assetHelper.withCleaner(wsk.trigger, triggerName) {
```

Comment:
this is the actual code change to ensure this test deletes the artifacts after it completes. everything else is just reformatting from scalariform